### PR TITLE
fix autoscaling description

### PR DESCRIPTION
**Pull request by qingwave**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
Which issue(s) this PR fixes:
Ref #99675

Special notes for your reviewer:
Does this PR introduce a user-facing change?
NONE
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

### CODE REVIEW

**Comment by qingwave** [comment]
cc @yuzhiquan

File: staging/src/k8s.io/api/autoscaling/v2beta2/types.go [hunk 1]
```diff
@@ -247,6 +247,7 @@ const (
 // ObjectMetricSource indicates how to scale on a metric describing a
 // kubernetes object (for example, hits-per-second on an Ingress object).
 type ObjectMetricSource struct {
+	// describedObject describes a kubernetes object
```
Comment (lavalamp) [inline staging/src/k8s.io/api/autoscaling/v2beta2/types.go:250]: This appears to be a reference, not a description. Please get a review from someone in the autoscaling SIG to confirm. Same comment for below change.

**Pull request closed without merging**